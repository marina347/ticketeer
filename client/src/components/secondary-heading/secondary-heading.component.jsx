import React from "react";
import "./secondary-heading.styles.scss";

export const SecondaryHeading = ({ text }) => {
  return <h1 id="secondary_heading" className="secondary-heading">{text}</h1>;
};

export default SecondaryHeading;
