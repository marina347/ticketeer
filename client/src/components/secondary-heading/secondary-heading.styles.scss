 .secondary-heading {
  text-transform: uppercase;
  color: var(--color-grey-dark-1);
  display: block;
  opacity: 0.7;
  font-size: 4.5rem;
  letter-spacing: 1rem;
  transition: all 0.2s ease-out;
  &:hover {
    transform: skew(2deg, 1deg);
  }
}
