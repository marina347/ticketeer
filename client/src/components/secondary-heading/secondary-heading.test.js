import React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import { SecondaryHeading } from "./secondary-heading.component";

describe("SecondaryHeading component", () => {
  let wrapper;
  let text;
  beforeEach(() => {
    text = "Text";
    wrapper = shallow(<SecondaryHeading text={text} />);
  });

  it("Should render SecondaryHeading component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("Should render secondary heading text", () => {
    const secondaryHeadingText = wrapper
      .find('[id="secondary_heading"]')
      .text();
    expect(secondaryHeadingText).toBe("Text");
  });
});
