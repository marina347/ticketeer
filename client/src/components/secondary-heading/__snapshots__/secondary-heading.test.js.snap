// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryHeading component Should render SecondaryHeading component 1`] = `
<h1
  className="secondary-heading"
  id="secondary_heading"
>
  Text
</h1>
`;
